compiler/fir/providers/src/org/jetbrains/kotlin/fir/scopes/FirOverrideService.kt

```diff
@@ -51,7 +47,7 @@ class FirOverrideService(val session: FirSession) : FirSessionComponent {
             val mostSpecific = selectMostSpecificMember(overridableGroup, returnTypeCalculator)
 
             overridableGroup.filterNotTo(conflictedHandles) {
-                isMoreSpecific(mostSpecific.member, it.member, returnTypeCalculator)
+                isMoreSpecificOrEqual(mostSpecific, it, returnTypeCalculator)
```

Comment:
The `selectMostSpecificInEachOverridableGroup` method is apparently not used anywhere. I'm not sure whether to

1. remove it completely;
2. rewrite it as
    ```
    if (members.size <= 1) return members
    val queue = LinkedList(members)
    val result = SmartSet.create<MemberWithBaseScope<D>>()
    while (queue.isNotEmpty()) {
        val nextHandle = queue.first()
        val overridableGroup = extractBothWaysOverridable(nextHandle, queue, overrideChecker)
        result.addAll(selectMostSpecificMembers(overridableGroup, returnTypeCalculator))
    }
    return result
    ```
    so that `isMoreSpecificOrEqual` and `selectMostSpecificMember` can be removed; or
3. just keep it as is for now.

Reply:
Removing is fine!